web/src/js/flow/utils.ts

```diff
@@ -287,6 +287,7 @@ export const getVersion = (flow: Flow): string => {
 export const sortFunctions = {
     tls: (flow: Flow) => flow.type === "http" && flow.request.scheme,
     icon: getIcon,
+    index: () => 0, // this is broken right now - ideally we switch to uuid7s on the backend and use that.
```

Comment:
what do you mean? why?

Reply:
Currently you can't sort by the index column, it's just a no-op. The problem is that we only have the flow as input here and not the index. Fixable, but not worth to complicate the API here for an off-by-default column. An easy fix would be to use uuid7s instead of uuid4s for `flow.id` and then sort by that, but that's future work. :)

Reply:
ahh I see, I forgot to try that 😅 okay! All good then :)